README.md

```diff
@@ -122,10 +122,19 @@ You can add additional variables using the `--bindings=BINDINGS` option. For exa
 
 
 ### Customizing behaviour with annotations
-
 - `kubernetes-deploy.shopify.io/timeout-override`: Override the tool's hard timeout for one specific resource. Both full ISO8601 durations and the time portion of ISO8601 durations are valid. Value must be between 1 second and 24 hours.
   - _Example values_: 45s / 3m / 1h / PT0.25H
   - _Compatibility_: all resource types (Note: `Deployment` timeouts are based on `spec.progressDeadlineSeconds` if present, and that field has a default value as of the `apps/v1beta1` group version. Using this annotation will have no effect on `Deployment`s that time out with "Timeout reason: ProgressDeadlineExceeded".)
+- `kubernetes-deploy.shopify.io/required-rollout`: Modifies how much of the rollout needs to finish
+before the deployment is considered successful.
+  - _Compatibility_: Deployment
```

Comment:
We should be consistent about how we reference objects and whether they're in backticks. The new uses both "ReplicaSet" and "`replicaSet`", and has "deployment" and "Deployment" and "deploy". The existing text above uses "`Deployment`".

Reply:
Good eye. I'm going to standardize on deployment and `replicaSet`